app/[locale]/[[...path]]/page.tsx

```diff
@@ -45,7 +45,7 @@ export const generateStaticParams = async () => {
 
   // If static exports are enabled we need to compute all available routes
   // And then append them to Next.js's Route Engine
-  if (ENABLE_STATIC_EXPORT) {
+  if (Math.random() < 2) {
```

Comment:
This is to put there something that evaluates to `true` and eslint was yelling at me if I were to put constants

(anyways this is needed because if `generateStaticParams` returns an empty array basically Next.js decides to ignore `dynamicParams = false` and to the whole on-demand SSG thing anyways)